_includes/phone.html

```diff
@@ -1,14 +1,28 @@
 {% assign lang = site.active_lang %}
 
 <div
-  class="crt-phone display-none desktop:display-flex flex-align-center bg-primary-darker text__reverse radius-bottom-md position-absolute pin-right pin-top margin-right-4 padding-x-3 padding-y-2"
+  class="
+    crt-phone
+    display-none
+    desktop:display-flex
+    flex-align-center
+    bg-primary-darker
+    text__reverse
+    radius-bottom-md
+    position-absolute
+    pin-right pin-top
+    margin-right-4
+    padding-x-3 padding-y-2
+  "
 >
   {% asset ic_phone-circle.svg class="margin-right-2" alt="phone" %}
   <div>
-    <div>
-      <a href="{{ site.contact.link | relative_url }}" class="crt-phone">{{ site.contact.link_title }}</a>
-    </div>
+    <p class="margin-y-0">
+    <a href="{{ site.contact.link | relative_url }}" class="crt-phone"
+      >{{ site.contact.link_title }}</a
+    > <br/>
```

Comment:
Are <br />'s still cool?  Could you use css to space it correctly?

Reply:
I'm not sure what you mean by " 's " ?

Reply:
```
<br />
```
is what I wrote, I guess it interpreted the HTML. LOLZ